index.html

```diff
@@ -1,12 +1,217 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Техномарт</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Техномарт</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <h1>
+        Техномарт
+      </h1>
+      <input type="search" placeholder="Поиск:">
+      <a href="blank.html">
```

Comment:
```suggestion
      <a href="blank.html">
```
Тег `a` ведет на страницу, которая указана в атрибуте `href`. Сейчас указано `blank.html`, но у нас такой страницы нету
Лучше сейчас, в таких местах, где у нас нету страниц, оставлять просто "якорь". Те - `href="#"`